source/sass/component/_select.scss

```diff
@@ -29,16 +26,15 @@ $c-select-border-radius-option: var(--c-select-border-radius-option, $border-rad
     margin: 0;
     outline: 0;
 
-    height: calc(#{$base}*6);
-    padding: map-get($spacers, "2");
+    height: map-get($spacers, "6");
+    padding: 0 map-get($spacers, "2");
+
     font-size: calc(#{$base}*2);
-    line-height: calc(#{$base}*2); 
-    border: 1px solid $color-form-fields;
+    line-height: 1; 
 
+    border: 1px solid $color-light;
     background: $color-form-fields;
-    background-size: calc(#{$base}*2);
-
-    border-radius: $c-select-border-radius;
+    border-radius: 2.5px;
```

Comment:
to keep things consistent and easy to change we probably want to create a variable for this (unless we already have one somewhere with this border radius?)

see: [source/sass/setting/_var_border-radius.scss](https://github.com/helsingborg-stad/styleguide/blob/master/source/sass/setting/_var_border-radius.scss)